pkg/runtime/gomemlimit_linux_nopython.go

```diff
@@ -0,0 +1,18 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux && !python
+
+package runtime
+
+import (
+	"context"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+func RunMemoryLimiter(c context.Context) error {
+	return NewStaticMemoryLimiter(config.Datadog.GetFloat64("gomemlimit_pct"), config.IsContainerized()).Run(c)
```

Comment:
We don't read anything anymore for `datadog.yaml` in `system-probe`? We can switch to `system-probe` config file for sure with a dedicated default if necessary.

Reply:
We do, but only for some limited CWS use cases and we are trying to limit/reduce it. This is because a non-root owned config file would then be controlling a root/privileged process.